bundles/org.eclipse.e4.ui.workbench.addons.swt/src/org/eclipse/e4/ui/workbench/addons/cleanupaddon/CleanupAddon.java

```diff
@@ -53,12 +54,16 @@ public class CleanupAddon {
 	@Inject
```

Comment:
`@Inject` needs to be removed. See related commit in dsg-4_11 https://github.com/Halliburton-Landmark/eclipse.platform.ui/commit/30ba28ccf9691474a04d7b6a74842a4492aa3368